requirements.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install the list of required packages
+  package: name={{ item }} state=installed
+  with_items:
+  - git
+  - rpm-ostree
+  - libvirt
+  - qemu-img
+
+- name: Download daemon rpm
+  get_url:
+    url: http://libslack.org/daemon/download/daemon-0.6.4-1.x86_64.rpm
+    dest: /tmp
+
+- name: Install daemon
+  command:
+    rpm --install /tmp/daemon-0.6.4-1.x86_64.rpm
+  ignore_errors: yes
```

Comment:
I considered proposing this, but sometimes it is a tradeoff to make. It is a single step that can be skipped with a tag in future... or do we want to be explicit and show each individual action. If you can make it work, please go ahead 👍